content/webapp/services/prismic/transformers/exhibitions.ts

```diff
@@ -19,14 +19,40 @@ import {
   parseTitle,
 } from '@weco/common/services/prismic/parsers';
 import { link } from './vendored-helpers';
-import { parseResourceTypeList } from '@weco/common/services/prismic/exhibitions';
 import { transformGenericFields, transformRichTextField, transformTimestamp } from '.';
 import { transformSeason } from './seasons';
 import { transformPlace } from './places';
 import { transformImagePromo, transformPromoToCaptionedImage } from './images';
 import { isNotUndefined } from '@weco/common/utils/array';
 import { isFilledLinkToDocumentWithData } from '../types';
 import { ExhibitionFormat } from '@weco/common/model/exhibitions';
+import { Resource } from '@weco/common/model/resource';
+
+// TODO: Use better types than Record<string, any>.
+//
+// This was lifted directly from a JavaScript implementation when we converted the
+// codebase to TypeScript (previously it was `Object`), but I can't find any exhibition
+// pages where we actually define/use any resources (or at least not any picked up by
+// the previous implementation), so I couldn't test it.
+function transformResourceTypeList(
+  fragment: Record<string, any>[],
+  labelKey: string
+): Resource[] {
+  return fragment
+    .map(label => label[labelKey])
+    .filter(Boolean)
+    .filter(label => label.isBroken === false)
```

Comment:
why the double filter?
```suggestion
    .filter(label => label && label.isBroken === false)
```

Reply:
> why the double filter?

Ask the original author of this code. 😄